Tools/Scripts/libraries/webkitapipy/webkitapipy/program.py

```diff
@@ -29,6 +29,7 @@
 import shlex
 import sys
 import time
+from fnmatch import fnmatch
```

Comment:
You never end up using `fnmatch` do you?

Reply:
Good catch. Cleaned up this and some other unused imports.